web/src/globalstyle.js

```diff
@@ -6,7 +6,7 @@ const FadeAnimationDefinition = css`
   .fade-exit { opacity: 1; }
   .fade-exit-active { opacity: 0; }
   /* Don't set display to 'none' to keep the same width and height */
-  .fade-exit-done { opacity: 0; pointer-events: none; }
+  .fade-exit-done { opacity: 0; pointer-events: none; visibility: hidden; }
```

Comment:
Make sure the element is hidden so we can use `await page.waitForSelector('#loading', { hidden: true });` in puppeteer